content/metadata.yaml

```diff
@@ -1387,3 +1387,20 @@ authors:
         order: 1
         contributions:
           - Writing - Review & Editing
+  - 
+    github: jessegmeyerlab
+    name: Jesse G. Meyer
+    initials: JGM
+    orcid: 0000-0003-2753-3926
+    email: jessegmeyer@gmail.com
+    affiliations:
+      - Department of Biochemistry, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America
+    coi:
+      string: "None"
```

Comment:
@jessegmeyerlab The only other thing I need from you is just any disclosure of conflicts of interest -- or just okay the "None" I have here!

Reply:
I do not have any conflicts of interest

Reply:
Sounds great! I'm merging it in now :)